### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
sir pls review and tell the mistakes

### CODE REVIEW

File: src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React, { Component } from 'react';
 import { FormGroup, Row, Col, Button, FormControl } from 'react-bootstrap';
-import Switch from 'react-flexible-switch';
+// import Switch from 'react-flexible-switch';
```
Comment (ShankyRana) [inline src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js]: remove commented code

File: src/components/NotificationSettings/NotificationConfigure/NotificationConfigure.scss [hunk 2]
```diff
@@ -386,3 +386,255 @@ img {
 .recent-settings .row, .bulk-settings .row {
       padding: 10px 0px;
 }
+
```
Comment (ShankyRana) [inline src/components/NotificationSettings/NotificationConfigure/NotificationConfigure.scss]: contain this scss

File: src/components/Notification/index.js [hunk 3]
```diff
@@ -92,12 +92,14 @@ class Notification extends Component {
         <td>{campaign.campaignName}</td>
         <td>{campaign.websiteUrl}</td>
         <td>
-          <Switch
+          {/* <Switch
```
Comment (ShankyRana) [inline src/components/Notification/index.js]: removed commented code

File: src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js [hunk 4]
```diff
@@ -349,7 +349,7 @@ export class NotificationDesignSetting extends Component {
                       <span className="mt-5 text-muted h6">Display map icon only</span>
                     </Col>
                     <Col md={2}>
-                      <Switch circleStyles={{
+                      {/* <Switch circleStyles={{
```
Comment (ShankyRana) [inline src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js]: remove comments

File: src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js [hunk 5]
```diff
@@ -526,7 +528,7 @@ export class NotificationDesignSetting extends Component {
                         </span>
                       </Col>
                       <Col md={2}>
-                        <Switch
+                        {/* <Switch
```
Comment (ShankyRana) [inline src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js]: here too

File: src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js [hunk 6]
```diff
@@ -566,7 +572,7 @@ export class NotificationDesignSetting extends Component {
                     <span className="mt-5 text-muted h6">Notifications Clickable</span>
                   </Col>
                   <Col md={2}>
-                    <Switch circleStyles={{
+                    {/* <Switch circleStyles={{
```
Comment (ShankyRana) [inline src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js]: here too

**Comment by ShankyRana** [comment]
Make the toggle scss global to dashboard if it is used multiple times

File: src/components/Dashboard/Dashboard.js [hunk 7]
```diff
@@ -4,7 +4,7 @@ import { browserHistory } from 'react-router';
 import { Row, Col } from 'react-bootstrap';
 import Moment from 'moment';
 import { extendMoment } from 'moment-range';
-import Circle from 'react-circle';
+// import Circle from 'react-circle';
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: remove

File: src/components/WebsitePricing/WebsitePricing.js [hunk 8]
```diff
@@ -160,12 +160,14 @@ class WebsitePricing extends Component {
             <div className="row justify-content-center">
               <div className="col-md-1 mr-0 text-left " id="leftmg"><div><strong onClick={this.handleMonthChange} className="h5 type--bold">Monthly</strong></div></div>
               <div className="col-md-1 col-sm-1 my-auto text-center pl-2">
-                <Switch
+                {/* <Switch
```
Comment (ShankyRana) [inline src/components/WebsitePricing/WebsitePricing.js]: remove

File: Component.jsx [hunk 9]
```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
```
Comment (ShankyRana) [inline Component.jsx]: What is this file for?

File: src/components/Profile/Profile.js [hunk 10]
```diff
@@ -59,6 +60,9 @@ class Profile extends Component {
   }
 
   updateProfile = (e) => {
+    this.props.loading ? console.log(this.props.loading)
```
Comment (ShankyRana) [inline src/components/Profile/Profile.js]: log..

Comment (GAUTAMRAJU15) [reply src/components/Profile/Profile.js]: removed this error .done

File: Component.jsx (hunk 9 above)
Comment (GAUTAMRAJU15) [reply Component.jsx]: i will delete that . removed

File: src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js (hunk 1 above)
Comment (Devilla) [reply src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js]: removed.

File: src/components/Notification/index.js (hunk 3 above)
Comment (Devilla) [reply src/components/Notification/index.js]: removed.

File: src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js (hunk 4 above)
Comment (Devilla) [reply src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js]: removed.

File: src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js (hunk 5 above)
Comment (Devilla) [reply src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js]: done

File: src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js (hunk 6 above)
Comment (Devilla) [reply src/components/NotificationSettings/NotificationConfigure/NotificationDesignSetting.js]: done.

File: src/components/Dashboard/Dashboard.js (hunk 7 above)
Comment (Devilla) [reply src/components/Dashboard/Dashboard.js]: done

**Pull request merged**